app/models/form/sales/subsections/property_information.rb

```diff
@@ -8,42 +8,40 @@ def initialize(id, hsh, section)
 
   def pages
     @pages ||= [
+      (uprn_questions if form.start_date.year >= 2024),
       Form::Sales::Pages::PropertyNumberOfBedrooms.new(nil, nil, self),
       Form::Sales::Pages::AboutPriceValueCheck.new("about_price_bedrooms_value_check", nil, self),
       Form::Sales::Pages::PropertyUnitType.new(nil, nil, self),
       Form::Sales::Pages::MonthlyChargesValueCheck.new("monthly_charges_property_type_value_check", nil, self),
       Form::Sales::Pages::PercentageDiscountValueCheck.new("percentage_discount_proptype_value_check", nil, self),
       Form::Sales::Pages::PropertyBuildingType.new(nil, nil, self),
-      uprn_questions,
-      postcode_and_la_questions,
-      Form::Sales::Pages::AboutPriceValueCheck.new("about_price_la_value_check", nil, self),
+      (uprn_questions if form.start_date.year == 2023),
+      (postcode_and_la_questions if form.start_date.year < 2023),
```

Comment:
Given the need for it for uprn_questions it made more sense to me to have all the year conditions done in the same way.

Reply:
Yeah, I think this is nicer!